eawx-build/Configuration/v1/CleanTaskBuilder.cs

```diff
@@ -5,7 +5,11 @@
 
 namespace EawXBuild.Configuration.v1 {
     public class CleanTaskBuilder : ITaskBuilder {
-        private readonly CleanTask _cleanTask = new CleanTask(new FileSystem());
+        private readonly CleanTask _cleanTask;
+
+        public CleanTaskBuilder(IFileSystem fileSystem) {
```

Comment:
Can that dependency be made optional by providing a default injection if the argument is `null`?
Something like
```csharp
public MyClass(IFileSystem fileSystem)
{
    m_fileSystem = fileSystem ?? new FileSystem();
}
```

That would make the "normal" creation - where we don't want to inject the mock file system - less verbose and more convenient.